### ISSUE TITLE
Unsupported Swift Version

### ISSUE THREAD

**Issue by BricePissard** [Kureev/react-native-uber-rides#2]
Installing the module with 
```
rnpm install react-native-uber-rides
```

or 
```
npm i react-native-uber-rides --save
react-native link react-native-uber-rides
```

Generate the XCode (v8.3.2) with deployment target 9+ 

The error:

> Xcode 8 will not be able to build the workspace and certain editing features will not function properly until the Swift source code in this workspace has been converted.
> This action can be performed later using “Convert to Current Swift Syntax” in the Edit menu.

After XCode have (intended) to convert the code into the new Swift version, the Swift code become red and buggy...

**_“Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly._**

![screen shot 2017-04-26 at 2 44 45 pm](https://cloud.githubusercontent.com/assets/4115874/25435037/ff15f12a-2a8e-11e7-9d5d-651c27409b29.png)

No way to fix it easily.

Any update require?

**Comment by Kureev** [comment]
Yeah, I assume it isn't compatible with the latest version of Swift lang. But as far as I'm not using this library anymore, I spend no time maintaining it. Although, if you manage to fix these issues and compose a PR, I'll be happy to merge it in.

**Comment by mackenziemance** [comment]
@BricePissard did you ever resolve this ??

**Comment by BricePissard** [comment]
@mackenziemance, sorry I ended not using this lib and left the Uber integration for later.

**Comment by andyprevalsky** [comment]
anyone able to get this working for swift 4?
